src/websocket-events/websocket-events.base.ts

```diff
@@ -50,6 +50,14 @@ export interface WebSocketMessage {
   data: any;
 }
 
+export interface WebSocketActionBase {
+  type: 'start' | 'ack' | 'nack' | 'protocol_error';
+}
+
+export interface WebSocketStart extends WebSocketActionBase {
```

Comment:
It looks like we have two competing spellings for websocket actions. The start action is spelled `{ "type": "start" }` while ack is spelled `{ "event": "ack" }`?

Reply:
It looks like the outgoing events have also retained the prior spelling, ie `{ "event": "started" }`.